tests/unit/async_db/test_typing_format.py

```diff
@@ -0,0 +1,72 @@
+from datetime import date, datetime, timedelta, timezone
+
+from pytest import raises
+
+from firebolt.async_db import DataError
+from firebolt.async_db._types import format_sql, format_value
+
+
+def test_format_value() -> None:
+    test_cases = (
+        # Strings
+        ("abcd", "'abcd'"),
```

Comment:
Any advantage over `@pytest.mark.parametrize` here?

Reply:
Good point, added it